app/endpoints/users.py

```diff
@@ -156,17 +156,17 @@ async def create_user_by_user(
     # Check the account type
 
     # For staff and student
-    # ^[\w\-.]*@(ecl\d{2})|(alternance\d{4})|(auditeur)|(master)?.ec-lyon.fr$
+    # ^[\w\-.]*@((etu(-enise)?|auditeur).)?ec-lyon.fr$
     # For staff
     # ^[\w\-.]*@(auditeur.)?ec-lyon.fr$
     # For student
-    # ^[\w\-.]*@((ecl\d{2})|(alternance\d{4})|(master)).ec-lyon.fr$
+    # ^[\w\-.]*@etu(-enise)?.ec-lyon.fr$
 
-    if re.match(r"^[\w\-.]*@(auditeur.)?ec-lyon.fr", user_create.email):
+    if re.match(r"^[\w\-.]*@(auditeur.)?ec-lyon.fr$", user_create.email):
         # Its a staff email address
         account_type = AccountType.staff
     elif re.match(
-        r"^[\w\-.]*@((ecl\d{2})|(alternance\d{4})|(master)).ec-lyon.fr$",
+        r"^[\w\-.]*@etu(-enise)?.ec-lyon.fr$",
```

Comment:
How about alternance ? Did their email changed too ?

Reply:
I think master are now considered as staff and alternant as students

Reply:
Masters are considered also as students, with `etu.ec-lyon.fr` email address